src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -78,9 +77,18 @@ object ProtocPlugin extends AutoPlugin {
 
       val protocCache = TaskKey[FileCache[ModuleID]]("protoc-cache", "Cache of protoc executables")
 
+      val cacheArtifactResolution = SettingKey[Boolean](
+        "cache-artifact-resolution",
```

Comment:
I followed the pattern of `cache-classloaders`, but I wondered if this should be prefixed with `protoc-` ?